src/change/sdp.rs

```diff
@@ -798,9 +798,17 @@ fn add_new_lines(
         if m.typ.is_media() {
             let mut media = Media::from_remote_media_line(m, idx);
             media.need_open_event = need_open_event;
+            // If this was for a sendonly m-line in the offer the Pts are merely a suggestion, we SHOULD attempt
+            // to match these.
+            // The check here is inverted because `Media::direction` is the direction from str0m's
+            // perspective
+            let params_suggestive =
```

Comment:
This isn't quite right, there are two conditions we need to consider here:

1. Is this a recvonly m-line(from the perspective of str0m)
2. Was it added in an offer(in which case it express the desire of the offerer).

Reply:
Got ya.